### PR TITLE
Added Flash Card and Card

### PR DESCRIPTION
**Pull request by Tanvez**
Need some help to do some styling to get it look like the mockups. This is for the Educational section.
ezgif-3-3afd7e14e490
ezgif-3-5236d8948ec6

Issue #137

### CODE REVIEW

File: client/src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { animated, interpolate } from "react-spring";
+import Button from '@material-ui/core/Button';
+
+class Card extends React.Component {
```
Comment (SomeMoosery) [inline client/src/components/Card.js]: Is there any reason this is a class component, and not functional? If not, let's try to keep things consistent as functional components as you've done with flashcards

Comment (Tanvez) [reply client/src/components/Card.js]: Will do!

Comment (SomeMoosery) [reply client/src/components/Card.js]: Looks good!

**Comment by SomeMoosery** [comment]
Looking good so far @Tanvez - just seems like there's still pesky merge conflicts that need rebasing. 

[This is a good place to start, let me know if you need any help!](https://stackoverflow.com/questions/15602037/git-rebase-upstream-master-vs-git-pull-rebase-upstream-master)

**Pull request merged**